students/17th/kangminji/js/westalogin.js

```diff
@@ -0,0 +1,65 @@
+const id =document.getElementById('id');
+const password = document.getElementById("password");
+const btn = document.getElementById('btn');
+
+
+
+
+
+
+
+password.addEventListener('keyup', function changcolor(event){
+
+const idvalue = document.getElementById('id').value;
+const passwordvalue = document.getElementById('password').value;
+
+if(idvalue&&passwordvalue){
+btn.style.backgroundColor =  'blue';
+
+}else {
+    btn.style.backgroundColor = ' rgb(185, 216,256)';
+}
+
+
+
+});
+
+id.addEventListener('keyup', function changcolor(event){
+
+    const idvalue = document.getElementById('id').value;
+    const passwordvalue = document.getElementById('password').value;
+    
+    
+    
+    if(idvalue&&passwordvalue){
+    btn.style.backgroundColor =  'blue';
+    
+    }else {
+        btn.style.backgroundColor = ' rgb(185, 216,256)';
+    }
+
+    
+    });
```

Comment:
changeColor라는 동일한 기능을 하는 함수를 두번 작성해주셨네요.
아래와 같이 함수를 선언한 뒤 동일한 함수를 호출해 같은 코드작성을 피할 수 있습니다. 
```suggestion
password.addEventListener('keyup', handleChangeColor);
id.addEventListener('keyup', handleChangeColor);

function handleChangColor(){
  const idvalue = document.getElementById('id').value;  
  const passwordvalue = document.getElementById('password').value;

  idvalue&&passwordvalue ?
    btn.style.backgroundColor =  'blue'
    : btn.style.backgroundColor = ' rgb(185, 216,256)'
};
```